### DISCUSSION TITLE
I hope you're successful in porting this to Windows

### DISCUSSION THREAD

**Discussion by yokhoe** [riotrah/browseratops#17] [General]
It's been a long wanted request of mine since I use Browsersaurus on my MBP and really like the UI and implementation.  I hope there is no hindrance with Windows and how it treats browser default politics.

**Comment by riotrah** [comment]
I'm actually very, very close. 

There are a couple of things I have left for a first release:

0. Fixing some quirks/bugs
1. Easy install/packaging
2. Most importantly, I cannot/have not yet figured out how to have an electron app like this one be set as the default browser (eg the default http/s link-type handler)

It might be as easy as registering the 'http'/'https' protocols within electron itself, or might require some more messing around with app manifests or something. Other projects that I tried to copy take advantage of proper windows app sdks in ways that this electron app cannot (I believe). In theory, I've been able to register this as the default handler for html _files_ but that doesn't really do anything. 

> **TBH i haven't actually tried much in a few weeks but having just seen this, I feel inspired to try now! **

Are you able or interested in contributing some time by any chance? It'll make it happen faster! Just need some bookkeeping and tracking of bugs, TODOs etc. Or even figuring out the default browser thing. 


But ofc no worries, I'm sure if you had the bandwidth you would have yourself in the first place haha.
